content/lxd/introduction.md

```diff
@@ -1,87 +1,116 @@
 
-# Integration with OpenNebula
-Starting from [OpenNebula EDGE](https://opennebula.org/get-your-hands-on-v-5-8-edge/) this cloud management platform packs official drivers to manage LXD compute nodes.
+### Application containers vs. system containers
+Application containers (as provided by, for example, Docker or Kubernetes) package a single process or application. System containers, on the other hand, simulate a full operating system and let you run multiple processes at the same time.
 
-Key features:
+Therefore, application containers are suitable to provide separate components, while system containers provide a full solution of libraries, applications, databases and so on. In addition, you can use system containers to create different user spaces and isolate all processes belonging to each user space, which is something you cannot do with application containers.
 
+![Application and system containers](/static/img/lxd/application-vs-system-containers.svg "Application and system containers")
 
+### Virtual machines vs. system containers
+Virtual machines emulate a physical machine, using the hardware of the host system from a full and completely isolated operating system. System containers, on the other hand, use the OS kernel of the host system instead of creating their own environment. If you run several system containers, they all share the same kernel, which makes them faster and more light-weight than virtual machines.
 
-* Enables **qcow2 backed containers** and regular KVM-like images, giving users more flexibility when deploying their workloads
-* Support for snap installed LXD nodes.
-* Ceph storage pools shared across all virtualization nodes (both KVM and LXD)
-* Seamless LXD public image servers integration in a marketplace-like style. Supports [LXD public image server](https://images.linuxcontainers.org),  [TurnkeyLinux](https://www.turnkeylinux.org), [DockerHub](https://hub.docker.com/) and also the [OpenNebula marketplace](https://marketplace.opennebula.systems/appliance)
-* [Multi-container services deployment](https://docs.opennebula.org/5.8/advanced_components/application_flow_and_auto-scaling/overview.html) with auto-scaling.
-* Heterogeneous hypervisors cloud deployments with KVM, LXD and VMWare vCenter managed by the same cloud orchestration system
+With LXD, you can create both system containers and virtual machines. You should use a system container to leverage the smaller size and increased performance if all functionality you require is compatible with the kernel of your host operating system. If you need a fully isolated environment or functionality that is not supported by the OS kernel of your host system, use a virtual machine.
+
+![Virtual machines and system containers](/static/img/lxd/virtual-machines-vs-system-containers.svg "Virtual machines and system containers")
+
+# Features
+Some of the biggest features of LXD are:
```

Comment:
Sry if this sounds quite negative, but I am not sure, whether the links to the documentation are such a good idea (in this place).

Let me explain:
I as a user would expect to be guided to either informative pages that detail these features or guides that really show you how to do it.
In contrast these pages are mostly either just a list of config keys or kind of sparse with information and very technical.
Some are also outdated.

So I would recommend to only use some of the very useful links for now and rework the documentation first for the others and then link to them.
Or add some more information directly here or on seperate page(s).

Reply:
We'll eventually get to work on those documentation pages, but website is where most of the work is currently needed.

What we could do maybe (I haven't seen the rendered website yet) would be to not have the entire sentence link but have a `(configuration details)` or the like which does link to the doc.

Reply:
The link is only on the main feature "name" or description or what you want to call it, not the whole line.

I agree some of the links are kinda ... loosely related. ;) But then, this shows what we need to focus on in the documentation.